doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,13 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-allow_ssl_eof>
+
+Some TLS implementation (such as L<openssl-s_server(1)> before the OpenSSL 3.0
+or IIS) do not send close_notify on shutdown.  Such behaviour is treated as
```

Comment:
I would drop the information in the parens here. Such things tend to get out-of-date very quickly.

s/implementation/implementations/

Reply:
Fixed.